packages/currencies/tests/index.test.js

```diff
@@ -169,6 +170,20 @@ test("formatShort", () => {
   expect(formatShort(getFiatUnit("EUR"), 123456)).toBe("1.2k");
 });
 
+test("chopCurrencyUnitDecimals", () => {
+  expect(chopCurrencyUnitDecimals(getFiatUnit("EUR"), "1")).toBe("1");
+  expect(chopCurrencyUnitDecimals(getFiatUnit("EUR"), "1,234")).toBe("1,234");
+  expect(chopCurrencyUnitDecimals(getFiatUnit("EUR"), "1,234.56")).toBe(
+    "1,234.56"
+  );
+  expect(chopCurrencyUnitDecimals(getFiatUnit("EUR"), "1,234.5678")).toBe(
+    "1,234.56"
```

Comment:
shouldn't it be `1,234.57` ?

Reply:
no. the method is called chop, not round ;)

it's meant for input text right? If I type `12.34` and then i add a `9` , it should just do nothing, not switch my input to `12.35`. and then `12.36` again if I type 9

Reply:
make sense :ok_hand: :